Firmware/ultralcd.cpp

```diff
@@ -2848,7 +2848,7 @@ bool lcd_calibrate_z_end_stop_manual(bool only_z)
                 lcd_encoder_diff = 0;
                 if (! planner_queue_full()) {
                     // Only move up, whatever direction the user rotates the encoder.
-                    current_position[Z_AXIS] += fabs(lcd_encoder);
+                    current_position[Z_AXIS] += lcd_encoder;
```

Comment:
Is this correct? Does the new code always move up? Because I have a feeling it won't preserve the old behaviour. Looks like it still needs an abs().